src/App.js

```diff
@@ -4,11 +4,16 @@ import AddTodoForm from './AddTodoForm';
 
 
 function App() {
+  const [todoList, setTodoList] = React.useState([]);
+    const addTodo = (newTodo) => {
+      setTodoList([...todoList, newTodo])
```

Comment:
Indentation for lines 8 and 9 isn't quite right, and there should be a `;` at the end of line 9.